### PR TITLE
chore: add color utils docs section

### PR DESCRIPTION
**Pull request by denieler**
[FX-1906](https://toptal-core.atlassian.net/browse/FX-1906)

### Description

Add a section for Color utils.

Screenshot 2021-05-13 at 11 48 44

<details>
<summary>PR commands</summary>
<br />

List of available commands:

- `@toptal-bot run all` - Run whole pipeline
- `@toptal-bot run build` - Check build
- `@toptal-bot run visual` - Run visual tests
- `@toptal-bot run deploy:documentation` - Deploy documentation
- `@toptal-bot run package:alpha-release` - Release alpha version

</details>

### CODE REVIEW

File: packages/picasso/src/utils/Utils/story/Colors.example.tsx [hunk 1]
```diff
@@ -0,0 +1,31 @@
+import { Table } from '@toptal/picasso'
+import React from 'react'
+
+const Example = () => {
+  const data = {
```
Comment (teimurjan) [inline packages/picasso/src/utils/Utils/story/Colors.example.tsx:5]: What do you think about extracting it out of the component's scope?

Comment (OleksandrNechai) [inline packages/picasso/src/utils/Utils/story/Colors.example.tsx:5]: I would inline this variable

**Review by OleksandrNechai — APPROVED** [review]

File: packages/picasso/src/utils/Utils/story/Colors.example.tsx (hunk 1 above)
Comment (denieler) [reply packages/picasso/src/utils/Utils/story/Colors.example.tsx:5]: I did it by analogy with https://github.com/toptal/picasso/blob/9c31dabb9bf0b9e34e8ac36b6add29237d43fb23/packages/picasso/src/utils/Utils/story/Generic.example.tsx#L5 and other utils. If we want to change - we would need to change everywhere, but I don't see huge reason to do that, so I would probably leave it as it is, ok?

File: packages/picasso/src/utils/Utils/story/Colors.example.tsx [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+    <Table>
+      <Table.Head>
+        <Table.Row>
+          <Table.Cell>Name</Table.Cell>
+          <Table.Cell>Description</Table.Cell>
+        </Table.Row>
+      </Table.Head>
+      <Table.Body>
+        {Object.entries(data).map(([name, description], index) => (
+          <Table.Row key={index}>
```
Comment (teimurjan) [inline packages/picasso/src/utils/Utils/story/Colors.example.tsx:21]: We have a rule not to use indices as keys. Let's use `name` as a key.

**Review by teimurjan — APPROVED** [review]

File: packages/picasso/src/utils/Utils/story/Colors.example.tsx (hunk 1 above)
Comment (OleksandrNechai) [reply packages/picasso/src/utils/Utils/story/Colors.example.tsx:5]: Yes, its rather nit, just this `data` and maps over it makes code a bit more complicated for not reason :-)

File: packages/picasso/src/utils/Utils/story/Colors.example.tsx (hunk 2 above)
Comment (denieler) [reply packages/picasso/src/utils/Utils/story/Colors.example.tsx:21]: yep, thank you 🙇‍♂️ the change is done 👍

File: packages/picasso/src/utils/Utils/story/Colors.example.tsx (hunk 1 above)
Comment (denieler) [reply packages/picasso/src/utils/Utils/story/Colors.example.tsx]: done 👍

**Pull request merged**